gui/test/stores/MemStoreSpec.scala

```diff
@@ -23,6 +23,14 @@ class MemStoreSpec extends StoreSpec with BeforeAndAfterAll {
       getMatchingNodesCount()
     }
 
+    "get total nodes count" in {
```

Comment:
There's a way for ScalaTest specs to inherit tests ("behave like") but I haven't figured out the incantation yet, thus the hacks.

Reply:
Ah well I think you've changed it, looks neat